### PR TITLE
Point users to Impeller from shader compilation warnings on iOS

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Screenshot 2023-03-20 at 1 29 23 PM

The note suggesting that users try Impeller will only show up when the connected app is running on iOS. The Impeller link takes users to the [wiki](https://github.com/flutter/flutter/wiki/Impeller).

Fixes https://github.com/flutter/devtools/issues/5349

CC @chinmaygarde @dnfield @jonahwilliams

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/performance/performance_utils.dart [hunk 1]
```diff
@@ -87,6 +87,8 @@ void debugTraceEventCallback(VoidCallback callback) {
 
 const preCompileShadersDocsUrl = 'https://docs.flutter.dev/perf/shader';
 
+const impellerWikiUrl = 'https://github.com/flutter/flutter/wiki/Impeller';
```
Comment (jonahwilliams) [inline packages/devtools_app/lib/src/screens/performance/performance_utils.dart:90]: @zanderso / @chinmaygarde are we keeping the wiki as the canonical source of documentation for Impeller?

**Review by jonahwilliams — APPROVED** [review]
LGTM with question

File: packages/devtools_app/lib/src/screens/performance/performance_utils.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/screens/performance/performance_utils.dart:90]: If there is a better link to point users to, let me know and I will change in a follow up PR. Thanks.

Comment (zanderso) [reply packages/devtools_app/lib/src/screens/performance/performance_utils.dart:90]: Yeah the wiki link is fine for now. If/when we shift to flutter.dev, we'll file an issue to update and also put a forwarding link in the wiki.

**Pull request merged**